content/developer/writing_plugins/packaging-your-plugin.md

```diff
@@ -37,12 +37,40 @@ When using Wagon version 0.3.2:
 $ wagon create -s path/to/python/project.zip -a '--no-cache-dir -c path/to/constraints.txt'
 ```
 
+When using Wagon version 0.11.0:
+ ```
+$ wagon create -r /path/to/reuierments/file/if/exits -v -f -a '--no-cache-dir -c path/to/constraints.txt' .
+```
+
+# Build py2py3 wagon
+
+Cloudify official plugins are python 2 and python 3 compatible. 
+In order to support installation of a wagon on both py2(5.0.5 and older) and py3 managers(5.1 and newer),
+py2py3 wagon introduced.
 
-When using Wagon version 0.6.1:
+py2py3 wagon is a wagon that contains all the python2 wheels and python3 wheels required for the plugin installation.
 
+From version 0.10.0 `--pip` flag introduced, when adding `--pip` flag to gagon command the wagon code use the specified `pip` in order to download the wheels.
+
+This way py2py3 wagons can be created.
+
+For example :
 ```
-$ wagon create path/to/python/project.zip -a '--no-cache-dir -c path/to/constraints.txt'
+$ wagon create --pip /env2/bin/pip --pip /env3/bin/pip --pyver "27" --pyver "36" -r /path/to/requierments/file -v -f -a '--no-cache-dir -c path/to/constraints.txt' path/to/plugin/source
 ```
 
+Where `env2` is python 2.7 virtualenv and `env3` is python 3.6 virtualenv.
```

Comment:
In this case `Where` is being used as an adverb to modify `env2` and `env3`. Since it is modifying them, it should be in the same sentence, not starting a new sentence. However, since the previous sentence is a text block, you should probably start a new sentence and replace `Where` with `Here`.